WordPress/src/main/res/values/strings.xml

```diff
@@ -970,6 +970,8 @@
     <string name="experimental_features_feedback_dialog_decline">Not now</string>
     <string name="experimental_subscribers_feature">Experimental subscribers</string>
     <string name="experimental_subscribers_feature_description">View and manage newsletter subscribers (work in progress)</string>
+    <string name="experimental_application_password_feature">Application Password log in</string>
+    <string name="experimental_application_password_feature_description">Enable Application Password to log-in self-hosted sites</string>
```

Comment:
Nit: I'm not wild about the double hyphenation with "log-in self-hosted sites", plus I think it would read better as "log into self-hosted sites."